plugins/module_utils/_api_data.py

```diff
@@ -113,16 +113,27 @@ def join_path(path):
             'arp-timeout': KeyInfo(default='auto'),
             'auto-negotiation': KeyInfo(default=True),
             'bandwidth': KeyInfo(default='unlimited/unlimited'),
+            'combo-mode': KeyInfo(can_disable=True),
+            'comment': KeyInfo(can_disable=True, remove_value=''),
             'disabled': KeyInfo(default=False),
+            'fec-mode': KeyInfo(can_disable=True),
             'full-duplex': KeyInfo(default=True),
             'l2mtu': KeyInfo(default=1598),
             'loop-protect': KeyInfo(default='default'),
             'loop-protect-disable-time': KeyInfo(default='5m'),
             'loop-protect-send-interval': KeyInfo(default='5s'),
             'mac-address': KeyInfo(),
+            'mdix-enable': KeyInfo(),
             'mtu': KeyInfo(default=1500),
             'name': KeyInfo(),
             'orig-mac-address': KeyInfo(),
+            'poe-out': KeyInfo(can_disable=True),
+            'poe-priority': KeyInfo(can_disable=True),
+            'poe-voltage': KeyInfo(can_disable=True),
+            'power-cycle-interval': KeyInfo(),
+            'power-cycle-ping-address': KeyInfo(),
```

Comment:
There is an "unset" command for power-cycle-ping-address, so should we add can_disable=True ? 
```suggestion
            'power-cycle-ping-address': KeyInfo(can_disable=True),
```

Reply:
Sounds good!